xilem_masonry/src/pod.rs

```diff
@@ -19,8 +19,8 @@ use crate::core::{Mut, SuperElement, ViewElement};
 ///
 /// [`transformed`]: crate::view::Transformed
 /// [`WidgetView::transform`]: crate::view::transformed
-#[expect(missing_docs, reason = "TODO - Document these items")]
 pub struct Pod<W: Widget + FromDynWidget + ?Sized> {
+    #[expect(missing_docs, reason = "Self-explanatory")]
```

Comment:
Don't be afraid of simple docs.

```suggestion
    /// A yet to be inserted widget.
```

Reply:
I honestly hate those when I read them.

I read so many docs where everything is like "`foobar_handler` - The handler that handles the foobars", YES, THANK YOU, I COULD INFER THAT FROM THE NAME, why are you wasting my time with this?

Reply:
> that the widget is not yet inserted, which might not be obivous?

That's what the first line of the type's doc says. (This is another trope of those "well the linter says we *have* to have a docstring" texts, they tend to restate what the parent doc already says.)